### ISSUE TITLE
mask of cross attention

### ISSUE THREAD

**Issue by nankepan** [NUS-HPC-AI-Lab/OpenDiT#115]
Valid word features are selected by mask at [https://github.com/NUS-HPC-AI-Lab/OpenDiT/blob/38d39325c88a8ac48fb5861fdcb9405b66cedf5c/opendit/models/opensora/stdit.py#L297](https://github.com/NUS-HPC-AI-Lab/OpenDiT/blob/38d39325c88a8ac48fb5861fdcb9405b66cedf5c/opendit/models/opensora/stdit.py#L297) However, attention mask becomes all -1e9 at [https://github.com/NUS-HPC-AI-Lab/OpenDiT/blob/38d39325c88a8ac48fb5861fdcb9405b66cedf5c/opendit/modules/attn.py#L342](https://github.com/NUS-HPC-AI-Lab/OpenDiT/blob/38d39325c88a8ac48fb5861fdcb9405b66cedf5c/opendit/modules/attn.py#L342) (m==y_lens). This seems to be incorrect because attention map will become all zeros after softmax.

**Comment by oahzxl** [comment]
thanks for your issue. we have fixed it now
